ui-lib/src/main/java/co/electriccoin/zcash/ui/common/Button.kt

```diff
@@ -0,0 +1,56 @@
+package co.electriccoin.zcash.ui.common
+
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.interaction.PressInteraction
+import androidx.compose.foundation.layout.RowScope
+import androidx.compose.material3.Button
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.seconds
+
+@Suppress("LongParameterList")
+@Composable
+fun TimedButton(
```

Comment:
May we move this new component into the ui-design-lib module to its Button class?